src/rpc/iterator/ExtGenFilm.tpp

```diff
@@ -0,0 +1,192 @@
+#ifndef RPC_EXT_GEN_FILM_TPP
+#define RPC_EXT_GEN_FILM_TPP
+
+/*
+* PSCF - Polymer Self-Consistent Field Theory
+*
+* Copyright 2016 - 2022, The Regents of the University of Minnesota
+* Distributed under the terms of the GNU General Public License.
+*/
+
+#include "ExtGenFilm.h"
+#include <prdc/cpu/RField.h>
+#include <pscf/math/IntVec.h>
+#include <util/containers/FArray.h>
+#include <rpc/field/FieldIo.h>
```

Comment:
Please include specific / high level classes and namespaces before generic / low level classes. For namespaces this means include in the order rpc, prdc, pscf, util.